cmd/iovnscli/extension/signature.go

```diff
@@ -0,0 +1,88 @@
+package extension
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/crypto/keys"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"io"
+	"os"
+)
+
+type SignatureSchema struct {
+	ChanID  string         `json:"@chain_id"`
```

Comment:
Please drop the `chain_id` as we discussed.  It will only add confusion since it's irrelevant for the signing of a message.